.github/scripts/review_agent.py

```
@@ -1,65 +1,27 @@
 import json
 import os
 import subprocess
 import sys
-import time
-from openai import OpenAI, RateLimitError
+import textwrap
+from pathlib import Path
+from openai import OpenAI
 
 PR_NUMBER = sys.argv[1] if len(sys.argv) > 1 else ""
 RESULT_FILE = "/tmp/review_result.json"
```

Ruff (0.15.12)

[error] Probable insecure usage of temporary file or directory: "/tmp/review_result.json"

(S108)